pynuodb/cursor.py

```diff
@@ -130,6 +130,22 @@ def execute(self, operation, parameters=None):
         if self.rowcount < 0:
             self.rowcount = -1
         self.rownumber = 0
+        
+    def executesqltest(self, test):
```

Comment:
Not yet.  It's a chicken and egg situation.  The nuodb changes are also
going in to execute the sql tests, once those are in I can add a test for
it.

On Thu, Aug 18, 2016 at 5:20 PM, Tom Gates <notifications@github.com> wrote:

> In pynuodb/cursor.py
> <https://github.com/nuodb/nuodb-python/pull/129#discussion_r75340204>:
>
> > @@ -130,6 +130,22 @@ def execute(self, operation, parameters=None):
> >          if self.rowcount < 0:
> >              self.rowcount = -1
> >          self.rownumber = 0
> > +
> > +    def executesqltest(self, test):
>
> Is there a test for this function? I don't see where executesqltest() is
> called.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nuodb/nuodb-python/pull/129/files/e17009a049cff9c29e38b0ed51e6ca9e212b3b9d#r75340204>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUGonETj5-BKnbASQN-lT76G4vqdtuxzks5qhIZNgaJpZM4JnT5N>
> .
>